.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ repos:
       - id: mixed-line-ending
       - id: trailing-whitespace
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.18.0
+    rev: v3.19.0
```

Comment:
Are these changes relevant to this PR? Would be nicer if these changes were separate.

Reply:
Had a vague error, turns out my pre-commit was very outdated. We don't really need to update these now. Reverted.